__tests__/RNColorPlalette-test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import 'react-native';
+import RNColorPalette from '../src/RNColorPalette';
+import renderer from 'react-test-renderer';
+
+test('renders correctly', () => {
+  const tree = renderer.create(<RNColorPalette />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+
+it('togglePalette', () => {
```

Comment:
The description of the test inside `it` should be very clear on what it should do.
For instance
`it('should change the state of the color picker when we toggle the palette', () => {})`

Reply:
description added.